core/src/test/java/org/mapfish/print/processor/map/CreateMapProcessorGridFixedNumlinesPointCRS84Test.java

```diff
@@ -47,7 +48,12 @@ public void testExecute() throws Exception {
     PJsonObject requestData = loadJsonRequestData();
     Values values =
         new Values(
-            "test", requestData, template, getTaskDirectory(), this.requestFactory, new File("."));
+            new HashMap<String, String>(),
```

Comment:
Maybe use AbstractMapfishSpringTest for new method.
Why "test" is no longer in Hashmap